share/spack/gitlab/cloud_pipelines/stacks/ml-darwin-aarch64-mps/spack.yaml

```diff
@@ -85,7 +85,7 @@ spack:
   ci:
     pipeline-gen:
     - build-job-remove:
-        tags: [ spack, public ]
+        tags: [ spack ]
```

Comment:
Should this go into the generic darwin configs?

Reply:
Yes, I think so. I updated this now.